### PR TITLE
Adding my Set Details

### PR DESCRIPTION
**Pull request by Alessandro-Benedetti**
When submitting a new or existing set details change, make sure you have followed the following steps:

### Adding New Set Details
- [x]  Add a new file (or update existing file) into the `set-details` or `set-details-polygon` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json). Make sure to provide the [checksum address](https://ethsum.netlify.app/) for your Set
- [x] Verify the JSON follows the correct schema according to this [template](https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json). You can verify it by pasting the schema into this [schema validator](https://www.jsonschemavalidator.net) and checking if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (see README for example)
- [x] If adding brand new token, make sure to add it to the `set.tokenlist.json` or `set.polygon.tokenlist.json` as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json

### Updating Existing Set Details
- [x] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [x] When generating a signature, make sure the version signed is for the one reflected in the version details for your entry.

### Signature (Required)

- [x] Attach the proper signature. You can generate one by going to the page for your Set on TokenSets and signing in. Tap on your Set's name and follow the instructions provided (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

0xa25c2ee2e668a5280af710a923b52e31a083a2d93c9e7c2e0e697ebe7801bf225b58045e8c2a5c53f7b20a5ac6ccd97ba42d851a28624e0482fc7be65039ebfb1c

### CODE REVIEW

File: set-details-polygon/0x3eC2000dC2AE3550AC6d82202E04a3993eb8510A.setdetails.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "Degen Bot",
+    "setSymbol": "DB",
+    "setDecimals": 18,
+    "setIcon": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/tokens/degen_bot.png",
```
Comment (justinkchen) [inline set-details-polygon/0x3eC2000dC2AE3550AC6d82202E04a3993eb8510A.setdetails.json:7]: not sure if you uploaded actual images. don't see anything loading when trying to view these

**Comment by Alessandro-Benedetti** [comment]
Maybe because the format of the picture is wrong. I try to fix it. Is the rest okay?

File: set-details-polygon/0x3eC2000dC2AE3550AC6d82202E04a3993eb8510A.setdetails.json (hunk 1 above)
Comment (Alessandro-Benedetti) [reply set-details-polygon/0x3eC2000dC2AE3550AC6d82202E04a3993eb8510A.setdetails.json]: Now it should be ok.

File: assets/managers/rakki_otoko.png [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (justinkchen) [inline assets/managers/rakki_otoko.png:1]: remove this file

File: assets/tokens/degen_bot.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (justinkchen) [inline assets/tokens/degen_bot.svg:1]: remove this file

**Comment by justinkchen** [comment]
mostly will be good once you remove those unused svg files

**Comment by justinkchen** [comment]
can you also add your token to the set.polygon.tokenlist.json file?

**Comment by Alessandro-Benedetti** [comment]
> can you also add your token to the set.polygon.tokenlist.json file?
sure, now I remove the files you indicated and add the token, thank you very much!

**Comment by Alessandro-Benedetti** [comment]
> > can you also add your token to the set.polygon.tokenlist.json file?
> > sure, now I remove the files you indicated and add the token, thank you very much!
it should be all right now, forgive my inexperience please! Thank you

**Comment by justinkchen** [comment]
Looks good thanks!

**Pull request merged**